cmd/skopeo/copy.go

```diff
@@ -196,6 +200,18 @@ func (opts *copyOptions) run(args []string, stdout io.Writer) error {
 			OciEncryptLayers:      encLayers,
 			OciEncryptConfig:      encConfig,
 		})
-		return err
+		if err != nil {
+			return err
+		}
+		if opts.digestFile != "" {
+			manifestDigest, err := manifest.Digest(manifestBytes)
+			if err != nil {
+				return err
+			}
+			if err = ioutil.WriteFile(opts.digestFile, []byte(manifestDigest.String()), 0644); err != nil {
+				return fmt.Errorf("failed to write digest to file %q: %w", opts.digestFile, err)
```

Comment:
```suggestion
				return fmt.Errorf("Failed to write digest to file %q: %w", opts.digestFile, err)
```

Reply:
Ah, I see this project uses capitalization for errors mostly, but https://github.com/golangci/golangci-lint doesn't like that.

Reply:
I think there’s a reasonable case for either lowercase or uppercase for library errors (and Go does standardize on lower case), but these errors show up directly in the UI as the top-level start of the sentence, where uppercase makes more sense. (We should get rid of the FATA prefix…)
```
FATA[0000] Failed to write digest to file …
```